packages/beacon-node/src/metrics/metrics/beacon.ts

```diff
@@ -71,6 +72,82 @@ export function createBeaconMetrics(register: RegistryMetricCreator) {
       help: "Current number of pending partial withdrawals",
     }),
 
+    eip7805: {
+      validInclusionListByteSize: register.counter({
+        name: "beacon_valid_inclusion_lists_size_bytes_total",
+        help: "Byte size of the inclusion list",
+      }),
+      invalidInclusionListByteSize: register.counter({
+        name: "beacon_invalid_inclusion_lists_size_bytes_total",
+        help: "Byte size of the inclusion list",
```

Comment:
this metric is questionable, why do we care about the size if they are invalid and rejected anyways?